wg-resource-management/README.md

```diff
@@ -11,7 +11,7 @@ To understand how this file is generated, see generator/README.md.
 Designing and shepherding cross-cutting features around compute resource isolation and utilization.
 
 ## Meetings
-* [Tuesdays at 18:00 UTC](https://zoom.us/j/4799874685) (weekly (On demand)). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC).
+* [Wednesday at 18:00 UTC](https://zoom.us/j/4799874685) (weekly (On demand)). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC).
```

Comment:
As per the agenda, this is correct. nit: s/Wednesday/Wednesdays
```
Meeting Information
Each Wednesday at 2PM EST / UTC-5 (18:00 UTC)
Web Conference: https://zoom.us/j/4799874685
```

And please don't modify this doc directly. You need to modify `sigs.yaml` in repo root and then run `make` to update this doc.